### PR TITLE
`content-visibility`: add compat features

### PR DESCRIPTION
**Pull request by ddbeck**
Some ideas for reviewing this:

- Are listed features plausible parts pieces of the feature as a whole?
- Are any of the listed features later additions, part of a distinct sub feature, or otherwise excludable?
- Are any pieces missing?

### CODE REVIEW

File: feature-group-definitions/content-visibility.yml [hunk 1]
```diff
@@ -1,2 +1,7 @@
 spec: https://drafts.csswg.org/css-contain-2/#content-visibility
 caniuse: css-content-visibility
+compat_features:
+  - api.ContentVisibilityAutoStateChangeEvent.ContentVisibilityAutoStateChangeEvent
+  - api.ContentVisibilityAutoStateChangeEvent.ContentVisibilityAutoStateChangeEvent.skipped
+  - api.ContentVisibilityAutoStateChangeEventk
```
Comment (tidoust) [inline feature-group-definitions/content-visibility.yml:6]:
```suggestion
  - api.ContentVisibilityAutoStateChangeEvent
```

**Review by tidoust — COMMENTED** [review]
Similar to #295, [CSS Containment Level 3](https://drafts.csswg.org/css-contain-3/#content-visibility) extends the definition of `content-visibility` to make it animatable. How would we convey that change?

**Comment by ddbeck** [comment]
> How would we convey that change?

I think there's two ways to look at it for this feature:

1. Since this feature is pre-interoperable, it's possible that it will be animatable across browsers by the time everyone has implemented the rest of it. If we think that's likely, then we can include the animation subfeature from BCD now (on the assumption that's what the `css.properties.content-visibility.keyframe_animatable` feature refers to; see https://github.com/mdn/browser-compat-data/pull/21120).

2. If we don't think animation will land with the rest of it, then we could create a follow-up feature for animation (e.g., `content-visibility-animation` or similar).

**Comment by tidoust** [comment]
I think I'm clearer now on the generic mechanism to handle semantic updates. That's good.

I don't feel strongly on whether we should do 1. or 2., but in the absence of more implementation insights, I would follow the CSS working group's own assumptions: level 2 should become stable before level 3. As such, if we want to talk about the feature sooner rather than later, we should probably not include more future-looking level 3 bits.

Once in a while, the assumptions will turn out to be wrong, but that's why they're called "assumptions" and not "assertions"...

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
I think that right now we don't need to assume anything about whether animation will be supported soon enough to be lumped into this feature. Down the line we'll most likely burn down BCD entries trying to match each to a feature, and then it will be clear if adding it into the `content-visibility` feature would change computed support or not.

**Pull request merged**